Client/Telemetry/TelemetryWrapper.swift

```diff
@@ -377,65 +377,59 @@ extension TelemetryWrapper {
         case tabView = "tab-view"
     }
 
-    public static func recordEvent(category: EventCategory, method: EventMethod, object: EventObject, value: EventValue, extras: [String: Any]? = nil) {
```

Comment:
This function was here to handle when the pref name is not in the 'extras' dict. Once it is in there, this function can go away.